.migration-backup/ui-components/src/services/class/__tests__/enrollment.test.ts

```
@@ -0,0 +1,275 @@
+/**
+ * Tests for ClassEnrollmentService
+ * Focuses on the enrollStudent method and its use of atomic enrollment
+ * Also includes tests for SQL injection prevention in getClassStudents
+ */
+
+import { ClassEnrollmentService } from '../enrollment';
+import { supabase } from '../../../utils/supabase';
```

ESLint

[error] '../../../utils/supabase' import is restricted from being used by a pattern. Avoid deep relative imports. Use path aliases (@ui, @domains, @lib, etc.) instead.

(no-restricted-imports)